client/src/app/routes.tsx

```diff
@@ -176,6 +177,10 @@ const buildAppRoutes = (
             key={keys.next().value}
             component={StardustAddressPage}
         />,
+        <Route path="/:network/nft/:address"
```

Comment:
Call the param `nftId` rather then `address`.

Reply:
I called it address so we can reuse AddressRouteProps in NftRedirecRout as props instad of adding NftRedirectRouteProps. wdyt?